src/rust/ide/src/double_representation/definition.rs

```diff
@@ -186,6 +165,22 @@ impl DefinitionName {
         };
         Some(DefinitionName {extended_target,name})
     }
+
+    /// Checks if the given definition name is a member of given type.
+    pub fn extends_type(&self, parent_name:&str, expected_typename:&str) -> bool {
```

Comment:
What does it mean? I udnerstand what docs tell, but I don't understand it in the context of Enso. Does it for example check if "Int" is member of "a" using our type set theory? Moreover, why the function is named "extends" ? We do not use "extend" word regarding our types in Enso. TBH I do not understand neither the docs, nor the connection between them and the fn name. I believe such docs should also have at least one line of example showing what they really mean.